easybuild/easyconfigs/w/waLBerla/waLBerla-6.1-foss-2021a.eb

```diff
@@ -0,0 +1,44 @@
+# easyconfig file for waLBerla v6.1
+easyblock = 'CMakeMake'
+
+name = 'waLBerla'
+version = '6.1'
+
+homepage = "https://walberla.net/index.html"
+description = """Widely applicable Lattics-Boltzmann from Erlangen is a 
+block-structured high-performance framework for multiphysics simulations"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://i10git.cs.fau.de/walberla/walberla/-/archive/v6.1']
+sources = ['%(name)s-v%(version)s.tar.gz']
+patches = ['waLBerla-6.1_fix_cmakelist_for_easybuild.patch']
+checksums = [
+    {'waLBerla-v6.1.tar.gz': 'f0acdd9ad6543bc9306c8aae953dd5065986271d4398916ae0469db8b21c007a'},
+    {'waLBerla-6.1_fix_cmakelist_for_easybuild.patch':
+     '13393d523d8f2e1856ae6acffc04bc74d35b5b6a7c800cfad75fb8e3678e4dad'},
+]
+
+
+builddependencies = [('CMake', '3.20.1')]
+
+dependencies = [
+    ('pybind11', '2.6.2'),
+    ('Python', '3.9.5'),
+    ('Boost.MPI', '1.76.0'),
+]
+
+parallel = 1
+
+configopts = "-DWALBERLA_BUILD_WITH_PYTHON=ON "
+configopts += "-DWALBERLA_BUILD_SHOWCASES=OFF "
+configopts += "-DWALBERLA_BUILD_DOC=OFF "
+configopts += "-DPython_ROOT_DIR=$EBROOTPYTHON "
+
```

Comment:
We need to set a PYTHONPATH in order for the python interface to be found. You can use:
```
modextrapaths = {'PYTHONPATH': ['walberla/pythonmodule/']}
```